tests/acceptance/bootstrap/AuthContext.php

```diff
@@ -216,6 +216,29 @@ public function userRequestsEndpointsWithNoAuthentication(string $method, TableN
 		}
 	}
 
+	/**
+	 * @When a user sends :method request to the following url with no authentication
+	 *
+	 * @param $method
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function aUserSendsRequestsOnTheFollowingUrlWithNoAuthentication($method, TableNode $table): void {
```

Comment:
Just matching the step-def.
```suggestion
	public function aUserSendsRequestToTheFollowingUrlWithNoAuthentication($method, TableNode $table): void {
```